src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -53,6 +55,14 @@ public class Controller
     private final static String DEFAULT_SCALING_PARAMETERS =
         CassandraRelevantProperties.UCS_SCALING_PARAMETER.getString();
 
+    /**
+     * The minimum sstable size. Sharded writers split sstables over shard only if they are at least as large as the
+     * minimum size.
+     */
+    static final String MIN_SSTABLE_SIZE_OPTION = "min_sstable_size";
+
+    static final long DEFAULT_MIN_SSTABLE_SIZE = FBUtilities.parseHumanReadableBytes(System.getProperty(PREFIX + MIN_SSTABLE_SIZE_OPTION, "100MiB"));
```

Comment:
All these properties must be retrieved through `CassandraRelevantProperties` (see e.g. `DEFAULT_SCALING_PARAMETER` above).